js/App.js

```diff
@@ -0,0 +1,68 @@
+var Socket = require('./utils/Socket'),
```

Comment:
Finally, bit the bullet and created a module where we could centralize functionality that is needed app-wide (or mostly app-wide).